code-challenges.js

```diff
@@ -45,16 +83,44 @@ var stringWithVowels2 = "ILoveJavaScript"
 var notAString1 = true
 var notAString2 = 42
 
+const deVowelizer = (string)  => {
 
+    if( typeof string === "string") {
+        return "true"
+    } else if (typeof string !== "string") {
+        return "not a string"
+    }
+    let vowels = ["a", "e", "i", "o", "u", "A", "E", "I", "O", "U"]
+    let newArry = string.split("")
+        
+    return newArry.filter(value => vowels.indexOf(value) === -1).join("")
+}
```

Comment:
cool! i'm totally following your logic here. however, your function can only run one return. as it is, if i pass the string "hello" to your function, my return is `true` instead of `"hll"`. there's some refactoring that needs to happen in order for this code to run. nothing major.